src/components/Header/Header.scss

```diff
@@ -28,14 +28,13 @@
     display: flex;
     justify-content: center;
     align-items: center;
-    padding: 1px 2px 0;
     font-size: 9px;
     font-weight: 700;
     background-color: $color-red;
     border-radius: 50%;
     position: absolute;
-    top: 18px;
-    right: 18px;
+    top: 17px;
+    right: 17px;
```

Comment:
1)need to change this value, because on Phones/Tablets/Desktops we have different sizes of link image
2 maybe a good choice add mixins and change offsets